ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -821,17 +822,20 @@ private void verifyResourcesAndPartitionEquivalencyInDc(Datacenter dc, String cl
       String partitionName = Long.toString(partition.getId());
       Set<String> replicaHostsInHelix = allPartitionsToInstancesInHelix.remove(partitionName);
       Set<String> expectedInHelix = new HashSet<>();
-      ensureOrThrow(replicaHostsInHelix != null, "No replicas found for partition " + partitionName + " in Helix");
-      for (Replica replica : partition.getReplicas()) {
-        if (replica.getDataNodeId().getDatacenterName().equals(dcName)) {
-          String instanceName = getInstanceName(replica.getDataNodeId());
-          expectedInHelix.add(instanceName);
-          ensureOrThrow(replicaHostsInHelix.remove(instanceName),
-              "Instance " + instanceName + " for the given replica in the clustermap not found in Helix");
-        }
+      List<Replica> replicaHostsInStatic = partition.getReplicas()
```

Comment:
Maybe we could rename the `replicaHostsInStatic` here, because it represents replicas not hosts in certain DC.

Reply:
Fixed !